docs/evaluate/how-it-works.mdx

```diff
@@ -0,0 +1,90 @@
+---
+id: how-it-works
+title: How Temporal Works
+sidebar_label: How It Works
+description: This guide provides a short overview of How Temporal Works.
+toc_max_heading_level: 4
+keywords:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+tags:
+  - temporal
+  - evaluate-temporal
+  - how-temporal-works
+---
+Temporal offers an entirely new way to build scalable and reliable applications.
+
+## Build Invincible Apps 
+
+Temporal ensures your applications are reliable by default, handling everything from minor network outages to significant issues like unexpected reboots or hardware failures.
+By offloading the responsibility of failure management from the application to the platform, this removes the need for extensive coding, testing, and maintenance tasks.
+
+This shift simplifies the development process and enables your application to seamlessly recover from disruptions.
+It does so by automatically reconstructing its previous state.
+Your application logic can be recovered, replayed, or paused from any previous point in the application process.
+
+![Describing a Temporal Application - How It Works](/img/how-it-works/hiw-yourapplicationclient.png)
+
+### Temporal Application
+
+A Temporal application is the code you write, comprised of Workflow and Activities. 
```

Comment:
```suggestion
A Temporal application is the code you write, composed of Workflow and Activities.
```

I changed it because "comprised of" sounded off to me, and then I googled it and found:

_Comprised of is an expression in English that means "composed of".[[1]](https://en.wikipedia.org/wiki/Comprised_of#cite_note-:0-1) This is thought by [language purists](https://en.wikipedia.org/wiki/Linguistic_purism) to be improper because to "comprise" (without the "of") can already mean to "be composed of"_